iaso/tasks/instance_reference_bulk_link.py

```diff
@@ -0,0 +1,85 @@
+from logging import getLogger
+from time import time
+from typing import List, Optional
+
+from django.contrib.auth.models import User
+from django.db import transaction
+
+from beanstalk_worker import task_decorator
+from hat.audit import models as audit_models
+from iaso.models import Instance, Task
+from iaso.models.org_unit import OrgUnitReferenceInstance
+from iaso.utils.models.common import check_instance_reference_bulk_link
+
+
+logger = getLogger(__name__)
+
+
+def link_single_reference_instance_to_org_unit(user: Optional[User], instance: Instance):
+    org_unit = instance.org_unit
+    form = instance.form
+    if not org_unit.reference_instances.filter(form=form).exists():
+        OrgUnitReferenceInstance.objects.create(instance=instance, org_unit=org_unit, form=form)
+    if org_unit.reference_instances.filter(id=instance.id).exists():
+        logger.info(f"updating {org_unit.name} {org_unit.id} by linking it to the instance  {instance.id}")
```

Comment:
I would not log inside an `if`, I would do it directly
```suggestion
    logger.info(f"updating {org_unit.name} {org_unit.id} by linking it to the instance  {instance.id}")
```